schema_utils.py

```diff
@@ -221,6 +240,17 @@ class _Validators(object):
     """
     @classmethod
     def get(cls, validator_id):
+        """Gets the value of the validator_id of the object.
```

Comment:
Isn't the validator ID being passed in?

I think you want something like "Gets the validation method corresponding to the given validator ID."

Reply:
Done